src/Exception/SerializableException.php

```diff
@@ -158,7 +158,7 @@ protected function filterTrace($trace, $file, $line)
      * @param  integer $level
      * @return array
      */
-    protected function filterArgs($args, $level = 0)
+    protected function filterArgs(array $args, int $level = 0): array
```

Comment:
Please do not add native php typehints in this PR as this introduces a BC break. 
Focus on PHP 8.0 compatibility only.
Thanks!